meta/plenta/contao-jobs-basic-zvoove-bundle/en.yml

```diff
@@ -0,0 +1,20 @@
+en:
+    title: zvoove Connector for Contao Jobs Basic
+    description: >
+        Jobs are periodically imported from zvoove, while applications are sent directly from Contao to the corresponding job ad in Zvoove.
```

Comment:
```suggestion
        Jobs are periodically imported from zvoove, while applications are sent directly from Contao to the corresponding job ad in zvoove.
```

Reply:
also @fritzmg

Reply:
@christianbarkowsky what do you mean here?